README.md

```diff
@@ -1 +1,13 @@
-# hw01
\ No newline at end of file
+# hw01: Self-Introduction
+My name is **Xuefei Gao** and you can just call me **Ian**, which is easier to pronounce and remember. I am a raising second year and I plan to major in Math and Econ. I am also a fellow of SISRM and I am going to work with Professor Susan Stokes this summer.
+
+![Photo](Self_Portrait.png)
+
+Here are some of my interests:
+1. Music
```

Comment:
I looked at the actual file and it seems like it had trouble processing the list within a list (i.e. the types of music you like). According to the Markdown guide (https://guides.github.com/features/mastering-markdown/), you need 4 spaces to indent your code, although it may also be possible that Markdown didn't like you using a bullet list within a numbered list. Despite this, I think it was a nice touch!